codespell_lib/data/dictionary.txt

```diff
@@ -57519,6 +57520,8 @@ validadion->validation
 validadions->validations
 validador->validator
 validadors->validators
+validaiton->validation, valuation,
```

Comment:
Have you found actual cases where `validaiton` stands for `valuation`?

Reply:
No. I have never seen a actual case, so I changed it to `validaiton->validation` only.